.github/workflows/translations.yml

```
@@ -10,7 +10,7 @@ on:
       - "**strings.json"
 
 env:
-  DEFAULT_PYTHON: "3.12.3"
+  DEFAULT_PYTHON: "3.11"
```

This should probably be updated to latest
```suggestion
  DEFAULT_PYTHON: "3.12"
```